onnx/test/hub_test.py

```diff
@@ -95,6 +95,10 @@ def test_get_model_info(self) -> None:
             AssertionError, lambda: hub.get_model_info("mnist", self.repo, opset=-1)
         )
 
+    def test_download_model_with_test_data(self) -> None:
+        dir = hub.download_model_with_test_data("mnist")
+        self.assertIsInstance(dir, str)
```

Comment:
Perhaps check more? For instance, check whether `test_data_set` and model exists under that directory.

Reply:
Sure, I thought that might be necessary, but was not sure which direction to proceed in.